model/task/task.go

```diff
@@ -562,6 +574,10 @@ func (t *Task) Reset() error {
 // Reset sets the task state to be activated, with a new secret,
 // undispatched status and zero time on Start, Scheduled, Dispatch and FinishTime
 func ResetTasks(taskIds []string) error {
+	catcher := grip.NewSimpleCatcher()
+	for _, id := range taskIds {
+		catcher.Add(testresult.RemoveByTaskIDAndExecution(id, 0))
+	}
```

Comment:
do we want to return early at this point?

Reply:
done